x-pack/plugins/security/public/plugin.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Plugin, CoreSetup } from 'src/core/public';
+import {
+  SessionExpired,
+  SessionTimeout,
+  SessionTimeoutHttpInterceptor,
+  UnauthorizedResponseHttpInterceptor,
+} from './session';
+
+export class SecurityPlugin implements Plugin<SecurityPluginSetup, SecurityPluginStart> {
+  public setup(core: CoreSetup, deps: {}) {
+    const { http, notifications, injectedMetadata } = core;
+    const { basePath, anonymousPaths } = http;
+    anonymousPaths.register('/login');
```

Comment:
I'm not sure if it could. These are "client side routes". For example. on the `/login` front-end route, any XHR requests to any endpoint, we ignore the 403s. I very well might be mistaken, but in the NP there won't be any server-side rendering, and the "client side routes" will be unknown on the server.

Reply:
Ah, I understand now -- thanks for explaining

Reply:
What we could do is have a option on the ApplicationService that specifies an app as "anonymous" and then use that to derive these paths.

Maybe we take a look at that after https://github.com/elastic/kibana/issues/41981 is complete for the login, logout, and status apps.